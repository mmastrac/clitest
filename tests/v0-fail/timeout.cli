#!/usr/bin/env clitest --v0

$ sleep 60
%TIMEOUT 100ms
