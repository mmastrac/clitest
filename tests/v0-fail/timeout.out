Running tests/v0-fail/timeout.cli ...

sleep 60
---
Process took too long!
tests/v0-fail/timeout.cli FAILED
Error: process took too long to join

